fe/fe-core/src/main/java/org/apache/doris/catalog/TableIf.java

```diff
@@ -561,4 +561,6 @@ default boolean isPartitionedTable() {
     }
 
     boolean autoAnalyzeEnabled();
+
+    TableIndexes getTableIndexes();
```

Comment:
not used in this pr, for future extension?

Reply:
getTableIndexes is used in this pr, one usage, 3 implementation. When columns has index, inferPredicate module will generate filter on this column.